pallets/bridge-mapping/src/mock.rs

```diff
@@ -1,63 +1,64 @@
-use crate::{Module, Trait};
+use crate::*;
 use sp_core::H256;
-use frame_support::{impl_outer_origin, parameter_types, weights::Weight};
+use frame_support::{parameter_types};
 use sp_runtime::{
-    traits::{BlakeTwo256, IdentityLookup}, testing::Header, Perbill,
+    traits::{BlakeTwo256, IdentityLookup}, testing::Header,
 };
 use frame_system as system;
+use crate as bridge_mapping;
 
-impl_outer_origin! {
-    pub enum Origin for Test {}
-}
+type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
+type Block = frame_system::mocking::MockBlock<Test>;
 
-// Configure a mock runtime to test the pallet.
+// For testing the pallet, we construct a mock runtime.
+frame_support::construct_runtime!(
+	pub enum Test where
+		Block = Block,
+		NodeBlock = Block,
+		UncheckedExtrinsic = UncheckedExtrinsic,
+	{
+		System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
+        BridgeMapping: bridge_mapping::{Pallet, Call, Config, Storage},
```

Comment:
Indentation.

Reply:
You mean like this ?
```
System: 	   frame_system::{Pallet, Call, Config, Storage, Event<T>},
BridgeMapping: bridge_mapping::{Pallet, Call, Config, Storage},
```

Reply:
Looks even weirder. Haha. Nevermind.

Reply:
We'll eal with the indentation when we reformat everything anyway